docs/examples/basic_meter/histogram.py

```diff
@@ -0,0 +1,65 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+This example shows how to use the different modes to capture metrics.
+It shows the usage of the direct, bound and batch calling conventions.
+"""
+import time
+
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import Measure, MeterProvider
+from opentelemetry.sdk.metrics.export.aggregate import HistogramAggregation, SummaryAggregation
+from opentelemetry.sdk.metrics.export import ConsoleMetricsExporter
+from opentelemetry.sdk.metrics.export.controller import PushController
+from opentelemetry.sdk.metrics.view import View
+
+# Use the meter type provided by the SDK package
+metrics.set_meter_provider(MeterProvider())
+meter = metrics.get_meter(__name__)
+exporter = ConsoleMetricsExporter()
+controller = PushController(meter=meter, exporter=exporter, interval=5)
+
+requests_size = meter.create_metric(
+    name="requests_size",
+    description="size of requests",
+    unit="1",
+    value_type=float,
+    metric_type=Measure,
+)
+
+hist_view = View(
+    requests_size,
+    HistogramAggregation(config=[0.2,0.4,0.6,0.8,1.0]),
```

Comment:
should config be renamed, or maybe be a more complex data structure that nests the histogram bounds? A top-level config that is just histogram bounds restricts future developers ability to provide further configuration to the aggregation. Having at least a dict means that one can add more fields without breaking backwards incompatibility.